besu/src/main/java/com/blockchaintp/besu/daml/DamlPublicPrecompiledContract.java

```diff
@@ -64,8 +64,9 @@ protected void handleTransaction(final LedgerState<DamlLogEvent> ledgerState, fi
       LOG.warn("InvalidProtocolBufferException when parsing log entry id", e1);
       throw new InvalidTransactionException(e1.getMessage());
     }
-    final SubmissionValidator<DamlLogEvent> validator = SubmissionValidator.create(ledgerState, () -> logEntryId, false,
-        Cache.none(), getEngine(), getMetricsRegistry());
+    final SubmissionValidator<DamlLogEvent> validator =
+      com.daml.ledger.validator.SubmissionValidator$.MODULE$.createForTimeMode(
```

Comment:
This might be your problem here actually the TimeMode method is what switches between wall clock etc.

Reply:
In case it helps with analyzing this diff, v1.13.1 had,
```scala
def create[LogResult](
    ledgerStateAccess: LedgerStateAccess[LogResult],
    allocateNextLogEntryId: () => DamlLogEntryId = () => allocateRandomLogEntryId(),
    checkForMissingInputs: Boolean = false,
    stateValueCache: StateValueCache = Cache.none,
    engine: Engine,
    metrics: Metrics,
): SubmissionValidator[LogResult] = {
  createForTimeMode(
    ledgerStateAccess,
    allocateNextLogEntryId,
    checkForMissingInputs,
    stateValueCache,
    engine,
    metrics,
    inStaticTimeMode = false,
  )
}
```